app/views/settings/_form.html.erb

```diff
@@ -15,11 +15,11 @@
   </div>
   <div class="form-field">
     <%= f.label :accepted_threshold %>
-    <%= f.number_field :accepted_threshold %>
+    <%= f.number_field :accepted_threshold, step: 0.01 %>
```

Comment:
I was thinking about `0.5` or `0.25`, but I guess now it does not matter as we decided to remove thresholds... ; ]